lib/helpers.js

```diff
@@ -25,24 +24,35 @@ exports.generateSignature = function generateSignature(dataToSign, secretKey) {
 };
 
 /**
- * Sign transaction.
+ * Generate method signature and pack it with encoded data.
  *
- * @param  {String} secretKey
- * @param  {Object} tx
- * @param  {String} chainId
- * @return {Object}
+ * @param  {String}   method Method ABI to get signature from.
+ * @param  {Object[]} values Values to encode.
+ * @return {String}          Hexadecimal tx data string with method signature and encoded values.
  */
-exports.sign = function sign(secretKey, tx, chainId) {
-    const encodedData = rlp.encode(JSON.stringify(tx));
-    const msgHash = keccak256(encodedData);
-    const sig = secp256k1.sign(msgHash, secretKey);
-    const recovery = sig.recovery;
+exports.encodeTxData = function encodeTxData(method, values) {
+    return values.length === 0
+        && '0x' + keccak256(method).toString('hex').slice(0, 8)
```

Comment:
Why 8? Should be a constant or description. Otherwise it's a magic number.

Reply:
4 bytes like default Ethereum method signature.